tutor/src/components/task-step/step-with-reading-content.cjsx

```diff
@@ -28,15 +28,15 @@ ReadingStepContent = React.createClass
 
   shouldOpenNewTab: -> true
   render: ->
-    {id, courseDataProps, stepType} = @props
+    {id, stepType} = @props
 
     {content_html, related_content} = TaskStepStore.get(id)
     {courseId} = Router.currentParams()
 
     <div className="#{stepType}-step">
-      <div
-        className="#{stepType}-content"
-        {...courseDataProps}>
+      <div className="#{stepType}-content">
+        <CourseTitleBanner courseId={courseId} />
```

Comment:
In this particular instance, and in the reference book, the course data props was being used to style the reading content/children of this element, and not a title banner:

![screen shot 2016-11-17 at 8 43 48 pm](https://cloud.githubusercontent.com/assets/2483873/20416819/c77a7246-ad06-11e6-9bfb-d0c29694ff7f.png)